src/openforms/forms/tests/test_copy.py

```diff
@@ -263,3 +264,35 @@ def test_copy_form_with_variables(self):
             1, variables_copy.filter(source=FormVariableSources.user_defined).count()
         )
         self.assertEqual(form_copy.formlogic_set.count(), 1)
+
+
+class CopyFormWithRegistrationBackendsTest(APITestCase):
```

Comment:
why does this have to be a new test case rather than a new test in the existing test case?

Reply:
No specific reason, I just saw that the above for variables was a different TestCase so I thought I should follow the pattern. I will update it and add it to the main one.

Reply:
that may have been because of the variables feature flag, feel free to merge all tests into the `CopyFormAPITests` test case, I don't see a reason why there should be multiple test cases